packages/dx-react-chart/docs/reference/legend.md

```diff
@@ -0,0 +1,58 @@
+# Legend Plugin Reference
+
+The Legend plugin allows to visualize legend.
+
+## Importing
+
+Use the following import statement:
+
+```js
+import { Legend } from '@devexpress/dx-react-chart-material-ui';
+// import { Legend } from '@devexpress/dx-react-chart-bootstrap4';
+```
+## User Reference
+
+### Properties
+
+Name | Type | Default | Description
+-----|------|---------|------------
+markerComponent | ComponentType&lt;Legend.MarkerProps&gt; | | A component that renders the marker.
```

Comment:
Where is the `MarkerProps` described?

Reply:
There are no specific props expect className in the markerComponent. So there is nothing to describe.

Reply:
OK. If we don't have an interface for this ComponentType, can we use `ComponentType<object>` type? We already use such a type for the [Table](https://devexpress.github.io/devextreme-reactive/react/grid/docs/reference/table/) plugin.